src/SCSSolverInterface.jl

```diff
@@ -160,6 +160,10 @@ status(m::SCSMathProgModel) = m.solve_stat
 getobjval(m::SCSMathProgModel) = m.obj_val
 getsolution(m::SCSMathProgModel) = m.primal_sol
 
+function invertsdconesize(p)
```

Comment:
You also have to make sure you're using SCS v1.1.0 or later, that includes the change for the lower triangular input format. (the seg fault is also worrying, although a separate issue).

The call to
```
invertsdconesize(p)
```
was to mimic the functionality that was there before, which was taking the square root. I'm not sure why the SCS.jl API expected the input cone to be n^2 rather than just n, so I remained consistent.